web/src/main/java/se/inera/intyg/rehabstod/logging/SickLeaveLogMessageFactory.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (C) 2023 Inera AB (http://www.inera.se)
+ *
+ * This file is part of sklintyg (https://github.com/sklintyg).
+ *
+ * sklintyg is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * sklintyg is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package se.inera.intyg.rehabstod.logging;
+
+import java.util.concurrent.TimeUnit;
+
+public class SickLeaveLogMessageFactory {
+
+    public static final String GET_ACTIVE_SICK_LEAVES = "GET_ACTIVE_SICK_LEAVES";
+    public static final String ADD_PATIENT_INFORMATION = "ADD_PATIENT_INFORMATION";
+
+    private long startTimer;
+
+    public SickLeaveLogMessageFactory(long startTime) {
+        startTimer = startTime;
+    }
+
+    public String message(String constant, int amount) {
+        return String.format("SICK LEAVE LOG - duration for %s: %d seconds. Amount: %d.", constant, timeElapsed(startTimer), amount);
```

Comment:
After reviewing the logs in the terminal, im wondering if the logs will look better if we use a capital D on "duration". Do you agree or should we just keep it as it is?

![MicrosoftTeams-image (3)](https://user-images.githubusercontent.com/57347249/231661625-ccb10d79-c37a-4936-bfff-bcc9c52e61e9.png)